src/Microsoft.FileFormats/Pointer.cs

```diff
@@ -113,13 +122,11 @@ public override string ToString()
             return instance.Value;
         }
 
-        internal void Init(ILayout targetLayout, ulong value)
+        public void Initialize(ulong value, PointerLayout layout)
```

Comment:
So the DCR can call in it's CustomPointerLayout class.  I guess it could go back to having it's own base CustomPointer base for the special "relative" pointers, but I chose to make this public.